partials/confirm_login.html

```diff
@@ -0,0 +1,51 @@
+<% extends "partials/base/base.html" %>
+
+<% block content %>
+<!--Header Area-->
+<tr style="page-break-before: always">
+  <td valign="top">
+    <p class="primary" style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0 0 24px 0; text-align: center;"><% block verify_content %>{{t "You signed in to Firefox from %(device)s, %(location)s."}} {{t "Please confirm this sign-in to begin syncing with %(device)." }}<% endblock %></p>
```

Comment:
@vbudhram - I'm checking the text against @ryanfeeley's mocks. Can you replace `Please confirm this sign-in to begin syncing with %(device).` with `For added security, please confirm this sign-in to begin syncing with %(device).`